x-pack/plugins/ml/public/application/timeseriesexplorer/components/forecasting_modal/run_controls.js

```diff
@@ -134,34 +137,60 @@ export function RunControls({
       <EuiForm>
         <EuiFlexGroup>
-          <EuiFlexItem>
-            <EuiFormRow
-              label={
-                <FormattedMessage
-                  id="xpack.ml.timeSeriesExplorer.runControls.durationLabel"
-                  defaultMessage="Duration"
-                />
-              }
-              fullWidth
-              isInvalid={!isNewForecastDurationValid}
-              error={newForecastDurationErrors}
-              helpText={
-                <FormattedMessage
-                  id="xpack.ml.timeSeriesExplorer.runControls.forecastMaximumLengthHelpText"
-                  defaultMessage="Length of forecast, up to a maximum of {maximumForecastDurationDays} days.
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup direction="column">
+              <EuiFlexItem grow={false}>
+                <EuiFormRow
+                  label={
+                    <FormattedMessage
+                      id="xpack.ml.timeSeriesExplorer.runControls.durationLabel"
+                      defaultMessage="Duration"
+                    />
+                  }
+                  fullWidth
+                  isInvalid={!isNewForecastDurationValid}
+                  error={newForecastDurationErrors}
+                  helpText={
+                    <FormattedMessage
+                      id="xpack.ml.timeSeriesExplorer.runControls.forecastMaximumLengthHelpText"
+                      defaultMessage="Length of forecast, up to a maximum of {maximumForecastDurationDays} days.
                   Use s for seconds, m for minutes, h for hours, d for days, w for weeks."
-                  values={{ maximumForecastDurationDays: FORECAST_DURATION_MAX_DAYS }}
-                />
-              }
-            >
-              {disabledState.isDisabledToolTipText === undefined ? (
-                durationInput
-              ) : (
-                <EuiToolTip position="right" content={disabledState.isDisabledToolTipText}>
-                  {durationInput}
-                </EuiToolTip>
-              )}
-            </EuiFormRow>
+                      values={{ maximumForecastDurationDays: FORECAST_DURATION_MAX_DAYS }}
+                    />
+                  }
+                >
+                  {disabledState.isDisabledToolTipText === undefined ? (
+                    durationInput
+                  ) : (
+                    <EuiToolTip position="right" content={disabledState.isDisabledToolTipText}>
+                      {durationInput}
+                    </EuiToolTip>
+                  )}
+                </EuiFormRow>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiFormRow
+                  helpText={i18n.translate(
+                    'xpack.ml.timeSeriesExplorer.runControls.neverExpireHelpText',
+                    {
+                      defaultMessage:
+                        'When not selected, created forecasts will expire after 14 days.',
```

Comment:
Not sure if `indefinitely` is technically correct. Is there an explicit hard limit for retaining forecasts? @peteharverson Let me know what you think, please.
```suggestion
                        'When selected, forecast results will be retained indefinitely.',
```